### DISCUSSION TITLE
下書き保存・公開設定の位置について改善要望

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#500] [要望・アイデア・不具合]
本日、エディタで記事を書く際の下書き保存・公開設定の位置について、従来の画面右下の位置から画面右上の位置に変更されていることに気づきました。

個人的には従来の位置の方が、使いやすかったです。
目の動きに合わせて、最後に右下を見て保存することも自然だと思います。

以上です。

**Comment by tanaka-asami415** [comment]
@Brutus03 
ご意見ありがとうございます。

https://qiita.com/release-notes/2023-07-11
2023/7/11に変わりました🙇
ヘッダーとフッターに表示されていた要素をまとめ、より執筆に利用できるエリアが広がるように変更いたしました。

長年下にあったものの位置を上部に変更したため、
従来の位置に慣れた方には、不便を感じさせてしまうということは理解しております🙇

様々なご意見を参考に、今後の改善に繋げていきたいと思います🙏
